src/drivers/bitbucket_cloud.js

```diff
@@ -21,9 +21,19 @@ class BitBucketCloud {
     const { project_path } = this;
     const { commit_sha, report } = opts;
 
-    // Print some variables
-    const endpoint = `/repositories/${project_path}/commit/${commit_sha}/comments/`;
-    const body = JSON.stringify({ content: { raw: report } });
+    // Get the PR ID #
+    const pr_endpt = `/repositories/${project_path}/commit/${commit_sha}/pullrequests`;
+    const pr_out = await this.request({ endpoint: pr_endpt });
+    const pr_id = pr_out.values[0].id;
```

Comment:
this will fail of there are no PRs associated with the sha. you need to check first if the lengh is bigger than 0

Reply:
```    if (!pr_out)
      throw new Error(
        'CML was unable to find a Pull Request corresponding to your commit.'
      );
```

how about that?

Reply:
@elleobrien It should do the comment also in the commit aside the comment in the PR. Right?
I did not even realize that the comment in the commit was missing. There will be guys that do not do PR's just only branches and they list the commits.
Throwing the exception is misleading. We need to additionally do the commit comment, thats exactly how GH works. When you do a comment in the commit is displayed in the PR comments but the comment is still in the commit. Thats what I wanted. To be able to show the comments in the PR aside to be in the commments.

Reply:
Ah, I see. So how about this:

- Put the comment in the commit, as before.
- If there is an associated PR, also but the comment in the PR.

Sound good?

Reply:
that sounds perfect!

Reply:
Latest commit should fix it. Now we write to the commit and then check if the PR ID query is valid- if so, write to PR. 

My biggest concern is if the if statement condition is foolproof, let me know what you think.